src/Orc.WorkspaceManagement/Services/Interfaces/IWorkspacesStorageService.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Orc.WorkspaceManagement.Services
```

Comment:
Remove .Services (we use a single namespace (Orc.WorkspaceManagement) for very simple packages).